Fixing translation and rtl

sovainfo opened the pull request:
Due to missing translation and issue with rtl changed the messages.
The suggested workaround didn't bring us the desired merge into the product. So, might as well do it as it should have been done!

davdebcom committed:
Update installation error to display source and destination folders #2

Tip found at https://www.rsjoomla.com/support/view-article/1012-troubleshooting-extension-installation-errors.html

davdebcom committed:
[CODE STYLE] Corrected concat space

Corrected: Concat operator must be followed by one space

davdebcom committed:
[CODE STYLE] Corrected concat space

davdebcom committed:
Update file.php

sovainfo committed:
Removed space before comma

sovainfo committed:
line 145 JLIB_FILESYSTEM_ERROR_COPY_FAILED

davdebcom committed:
Merge pull request #1 from sovainfo/patch-24

Removed space before comma

sovainfo committed:
Adding sprintf, sorry

davdebcom committed:
Merge pull request #2 from sovainfo/patch-24

Adding sprintf, sorry

sovainfo committed:
Fixing translation and rtl

sovainfo committed:
Fixing translation and rtl

sovainfo committed:
Fixing translation and rtl

sovainfo committed:
Fixing translation and rtl

sovainfo committed:
Fixing translation and rtl

infograf768 commented:
May I suggest to create a new string instead of using the older one?
The reason is that if a language pack is not updated, the new error display will not show at all.
If we use a new string, then at least en-GB would be used.
Also, please use %1$s and %2$s (instead of %s ) to let the Translation team order as wished.

sovainfo committed:
Workaround for sprintf

Created separate messages because of introducing sprintf and used sequense numbers for variables so translators can order the arguments

sovainfo committed:
Workaround for sprintf

Created separate messages because of introducing sprintf and used sequense numbers for variables so translators can order the arguments

sovainfo committed:
Workaround for sprintf

Created separate messages because of introducing sprintf and used sequense numbers for variables so translators can order the arguments

sovainfo committed:
Update en-GB.lib_joomla.ini

infograf768 commented:
Still :
>Also, please use %1$s and %2$s (instead of %s ) to let the Translation team order as wished.

Also, this PR has conflicts.
